dev/com.ibm.ws.microprofile.config.1.4/resources/com/ibm/ws/microprofile/config14/resources/Config14.nlsprops

```diff
@@ -0,0 +1,37 @@
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWMCG
+#COMPONENTNAMEFOR CWMCG MicroProfile Config API 
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#ISMESSAGEFILE true
+# #########################################################################
+###############################################################################
+# Copyright (c) 2018 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# This file follows the WebSphere Message Guidelines.
+# For more information, visit: 
+# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
+#
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Messages used by Config 1.1 Impl -  range is 0000-0099, plus 9999 for the emergency message
+# Messages used by Config 1.2 Impl -  range is 0100-0199, plus 9998 for the emergency message
+# Messages used by Config 1.3 Impl -  range is 0200-0299, plus 9997 for the emergency message
+# Messages used by Config 1.4 Impl -  range is 0300-0399, plus 9996 for the emergency message
+#-----------------------------------------------------------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------------------------------------------------------
+# Emergency Config 1.4 error message
+#-----------------------------------------------------------------------------------------------------------------------------
+
+# A Config API error has occurred: {0}
+temporary.CWMCG9996E=CWMCG9996E: A Config API error has occurred: {0}.
+temporary.CWMCG9996E.explanation=A Config API error has occurred that prevents normal operation.
+temporary.CWMCG9996E.useraction=Review the server message.log and FFDC logs to identify the problem.
```

Comment:
Suggestion:
Review the server message.log file and FFDC logs to identify the problem.

Are there any other hints or tips that we can give the users about what might be wrong?

Reply:
Thank you Brandy, I will include these changes when I get back to working on this PR. I can't think what other hints and tips we could give ... hopefully this message will never actually get used; it is for emergency purposes only. Anything else should have it's own bespoke message.